addons/sale_stock/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,49 @@
+---Fields in module 'sale_stock'---
+sale_stock   / procurement.group        / sale_id (many2one)            : NEW relation: sale.order
+# Filled in post-migration script
+
+sale_stock   / sale.order               / picking_ids (many2many)       : not a function anymore
+sale_stock   / sale.order               / picking_ids (many2many)       : type is now 'one2many' ('many2many')
+# Nothing to do
+
+sale_stock   / sale.order               / procurement_group_id (many2one): previously in module sale
+# Done, rename references on sale pre-migration script
+
+sale_stock   / sale.order.line          / move_ids (one2many)           : NEW relation: stock.move
+# Nothing to do
+
+sale_stock   / stock.move               / sale_line_id (many2one)       : NEW relation: sale.order.line
+# Filled in post-migration script
+
+sale_stock   / stock.move               / to_refund_so (boolean)        : DEL
+# Nothing to do
```

Comment:
Map to to_refund if stock_account is installed. You may have to create the column as the module loading order seems undefined to me.